ps/GetUmbracoBuildEnv.ps1

```diff
@@ -222,7 +222,7 @@ $ubuild.DefineMethod("GetUmbracoBuildEnv",
     $msBuild = $null
     $toolsVersion = ""
 
-    $vsMajor = if ($option.VsMajor) { $option.VsMajor } else { "15" } # default to 15 (VS2017) for now
```

Comment:
That is probably fine. I just cannot remember why I set 15 as default. Maybe during VS2019 preview phase.

Reply:
We were trying to figure this out this afternoon and couldn't find the reason either, however it would (in the future) be a bit more flexible if we just took the highest available VS version for the default and used `$option.VsMajor` instead, if it is being passed in.

Reply:
FYI, this still works on a machine where VS2019 is not available!